freeciv-web/src/main/webapp/javascript/control.js

```diff
@@ -1313,10 +1309,6 @@ Return a pointer to a visible unit, if there is one.
 **************************************************************************/
 function find_visible_unit(ptile)
 {
-  var panyowned = null;
-  var panyother = null;
-  var ptptother = null;
-  var pfocus = null;
```

Comment:
This one is used, please add var to the assignment some ten lines below.

Reply:
fixed